packages/token-bridge-sdk/src/deposits/fetchDeposits.ts

```diff
@@ -0,0 +1,112 @@
+import { Provider } from '@ethersproject/providers'
+import { AssetType } from '../hooks/arbTokenBridge.types'
+import { Transaction } from '../hooks/useTransactions'
+import { utils } from 'ethers'
+import { updateAdditionalDepositData } from '../util/deposits'
+import {
+  fetchDepositsFromSubgraph,
+  FetchDepositsFromSubgraphResult
+} from './fetchDepositsFromSubgraph'
+
+export type DepositETHSubgraphResult = {
+  id: string
+  senderAliased: string
+  destAddr: string
+  value: string
+  msgData: string
+  transactionHash: string
+  blockCreatedAt: string
+}
+
+/* Fetch complete deposits - both ETH and Token deposits from subgraph into one list */
+/* Also fills in any additional data required per transaction for our UI logic to work well */
+/* TODO : Add event logs as well */
```

Comment:
NIT: can we create an issue/notion ticket to handle this later?

Reply:
Checking if we need to ship this feature in the current drop. If not, then will create a ticket.

Reply:
Created an issue : https://github.com/OffchainLabs/arb-token-bridge/issues/598